cpp/include/cudf/io/config_utils.hpp

```diff
@@ -49,5 +50,16 @@ bool is_all_enabled();
  */
 bool is_stable_enabled();
 
-}  // namespace io::nvcomp_integration
+}  // namespace nvcomp_integration
+
+namespace kvikio_setting {
+
+/**
+ * @brief Set kvikIO thread pool size according to the environment variable KVIKIO_NTHREADS. If
+ * KVIKIO_NTHREADS is not set, use 8 threads by default.
+ */
+void set_thread_pool_nthreads_from_env();
```

Comment:
IMO this could be under cufile_integration

Reply:
Completed.